src/controllers/user.js

```diff
@@ -84,10 +84,13 @@ class UserController {
 
   async view(req, res) {
     try {
-      if (req.user.id !== parseInt(req.params.id, 10)) {
-        return res.status(403).json({ message: 'Access denied' });
-      }
-      const user = await User.findByPk(req.params.id);
+      // TODO: @lana: please change the rest of the backend to use email base instead of user ID. provided working example as follows :
```

Comment:
The app is used email to log in. The backend here uses id. So it is not fixed, the rest of the app wouldn't work